### PR TITLE
chore: update unit test for string ID types

### PR DESCRIPTION
**Pull request by jkaster**
Unit test tweak to support Looker 22.4 API type changes

### CODE REVIEW

File: packages/sdk-codegen-scripts/src/exampleMiner.spec.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ describe('example mining', () => {
     it('getRemoteHttpOrigin', () => {
       const actual = getRemoteHttpOrigin()
       expect(actual).toEqual(
-        'https://github.com/looker-open-source/sdk-codegen'
+        'https://github.com/looker-open-source/sdk-codegen.git'
```
Comment (joeldodge79) [inline packages/sdk-codegen-scripts/src/exampleMiner.spec.ts:78]: CI doesn't like this: [action output](https://github.com/looker-open-source/sdk-codegen/runs/5547364560?check_suite_focus=true)

File: packages/sdk-node/test/methods.spec.ts [hunk 2]
```diff
@@ -754,7 +754,7 @@ describe('LookerNodeSDK', () => {
           const query = await sdk.ok(sdk.create_query(request))
           expect(query).toBeDefined()
           expect(query.id).toBeDefined()
-          expect(query.id).toBeGreaterThan(0)
+          expect(Number(query.id)).toBeGreaterThan(0)
```
Comment (joeldodge79) [inline packages/sdk-node/test/methods.spec.ts:757]: hm, now that IDs really are strings, maybe

```suggestion
          expect(query.id.length).toBeGreaterThan(0)
```

**Review by joeldodge79 — CHANGES_REQUESTED** [review]

File: packages/sdk-node/test/methods.spec.ts (hunk 2 above)
Comment (jkaster) [reply packages/sdk-node/test/methods.spec.ts]: TypeScript didn't like that so I'm using a `String()` cast

File: packages/sdk-codegen-scripts/src/exampleMiner.spec.ts (hunk 1 above)
Comment (jkaster) [reply packages/sdk-codegen-scripts/src/exampleMiner.spec.ts]: modified the test statement

**Pull request merged**